.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
Aren't the composer-installed? Just like our own shell scripts are? That would explain why they aren't in the core teams bin folder, but they are in ours (if we started by doing a `create-project` from them.) We're essentially commit the artifacts put there by the cakephp/cakephp composer.json, if my guess is correct here.